meta-oe/recipes-dbs/timescaledb/timescaledb.inc

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "TimescaleDB is a relational database for time-series data built as an extension for PostgreSQL."
+HOMEPAGE = "https://timescale.com"
+LICENSE = "APACHE-2.0 & TIMESCALE"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/timescaledb-${PV}/LICENSE;md5=a6919544b6c76fc2e0debe84e8dfc7a5 \
+                    file://${WORKDIR}/timescaledb-${PV}/LICENSE-APACHE;md5=2ee41112a44fe7014dce33e26468ba93 \
+                    file://${WORKDIR}/timescaledb-${PV}/tsl/LICENSE-TIMESCALE;md5=1b3b062af96cae618d6ebd31df2712c8 \
+"
+
+DEPENDS = "postgresql"
+RDEPENDS_${PN} = "postgresql"
+
+SRC_URI = "git://git@github.com/timescale/timescaledb.git;protocol=ssh;branch=${BRANCH};tag=${PV};destsuffix=timescaledb-${PV}"
```

Comment:
this seems to be r/w access, wont work for anonymous clone, secondly use hardcoded SHA for SRCREV, using tags will require network access on build.